.cirrus.yml

```diff
@@ -0,0 +1,9 @@
+freebsd_instance:
+  image: freebsd-12-0-release-amd64
+
+task:
+  install_script: pkg install -y
+    autoconf bash binutils coreutils e2fsprogs-libuuid
+    git gmake libunwind ncurses pkgconf
+
+  script: CFLAGS="-Wno-unused-value" gmake
```

Comment:
Do you still need CFLAGS? If yes, why?

What about tests? Are there any issues left? We can enable it later if you want to just get this PR in.

Reply:
without CFLAGS I get:
```
cc -MD -c -o ../nondebug/libpmem/file_posix.o -std=gnu99 -Wall -Werror -Wmissing-prototypes -Wpointer-arith -Wsign-conversion -Wsign-compare -Wconversion -Wunused-macros -Wmissing-field-initializers -Wunreachable-code-return -Wmissing-variable-declarations -Wfloat-equal -Wswitch-default -O2 -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=2 -std=gnu99 -fno-common -pthread -DSRCVERSION=\"1.6+git28.g324a5af94\"   -Ix86_64 -DAVX512F_AVAILABLE=1 -I. -I../include -I../common/ -I../../src/../src/freebsd/include -I/usr/local/include -fPIC  ../../src/../src/common/file_posix.c
../../src/../src/common/file_posix.c:38:9: error: macro is not used [-Werror,-Wunused-macros]
#define _GNU_SOURCE
        ^
1 error generated.
gmake[2]: *** [../Makefile.inc:330: ../nondebug/libpmem/file_posix.o] Error 1
```

I will iterate on the tests soon, I just wanted to get this PR in first.